apps/server/lib/lexical/server/code_intelligence/completion/builder.ex

```diff
@@ -126,6 +127,11 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Builder do
       [{:atom, token, _}] ->
         length(token) + 1
 
+      [{:eol, '\n', []}] ->
```

Comment:
so, another option would be to change `prefix_length` to use `Code.Fragment` rather than the tokenizer, though i'm a little worried about that since I don't think this code is exhaustively tested.